app-backend/db/src/test/scala/com/azavea/rf/database/UserDaoSpec.scala

```diff
@@ -443,4 +443,25 @@ class UserDaoSpec
       }
     }
   }
+
+  test("get list of user ids and emails") {
+    check {
+      forAll { (userCreates: NonEmptyList[User.Create]) =>
+        {
+          val listIO = for {
+            _ <- userCreates traverse { userCreate =>
+              UserDao.create(userCreate)
+            }
+            listed <- UserDao.getThinUsersForIds(userCreates.map(_.id).toList)
```

Comment:
similarly to above, this nel -> list -> nel dance isn't really necessary if the function doesn't act like it wants to do work to empty lists as well